parl/remote/tests/remote_test.py

```diff
@@ -277,6 +283,21 @@ def test_thread_safe_of_remote_module(self):
         for t in threads:
             t.join()
 
+    def test_remote_object_with_call_raise_exception_function(self):
+        server_port = 17781
+        self._setUp(server_port)
+
+        remote_sim = self.remote_manager.get_remote()
+
+        try:
+            remote_sim.will_raise_exeception_func()
+        except RemoteError as e:
+            logger.info('Expected exception: {}'.format(e))
+            # expected
```

Comment:
To assure trackback information has been captured and sended to the server, you should assert that `Traceback`  is in the error string.

Reply:
done